core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -51,6 +51,15 @@ private[spark] abstract class WebUI(
   extends Logging {
 
   protected val tabs = ArrayBuffer[WebUITab]()
+  protected val initHandler: ServletContextHandler = {
+    val servlet = new HttpServlet() {
+      override def doGet(req: HttpServletRequest, res: HttpServletResponse): Unit = {
+        res.setContentType("text/html;charset=utf-8")
+        res.getWriter.write("Spark application is starting, please wait for start up.")
```

Comment:
I prefer the original hint.
Let's make it 
```
Spark is starting up. Please wait a while until it's ready.
```

Reply:
Done